bin/rhts-recipe-sync-set

```diff
@@ -60,20 +72,20 @@ def sync_set(state):
                 # We successfully set our state.  Exit.
                 break
         except:
-            print "Unable to connect to server, sleeping 5 seconds..."
+            print("Unable to connect to server, sleeping 5 seconds...")
             time.sleep(5)
 
     if(resp != 0) :
-        raise NameError, "ERROR: setting state...."
+        raise NameError("ERROR: setting state....")
 
 def usage():
-    print USAGE_TEXT
+    print(USAGE_TEXT)
     sys.exit(-1)
 
 def main():
     global result_server, hostname, recipesetid
 
-    if ('RESULT_SERVER' in os.environ.keys()):
```

Comment:
Are you sure you intended to remove 'in'?  I'm thinking you wanted to remove 'list' from previous changes and took in with you.  You didn't do it for RECIPESETID and HOSTNAME?????

Reply:
Yes, thanks, it was a mistake...... Will fix it.....